tests/tests_app/runners/test_singleprocess.py

```diff
@@ -1,3 +1,8 @@
+import os
```

Comment:
do we even need to keep the single process runtime anymore?

Reply:
No, I think it could be removed